source/elements/oneDPL/source/parallel_api/parallel_range_api.rst

```diff
@@ -49,6 +50,33 @@ of parallel range algorithms.
    template<typename I, typename Proj>
    using /*projected-value-type*/ = std::remove_cvref_t<std::invoke_result_t<Proj&, std::iter_value_t<I>&>>;
 
+  // Extension of C++20 exposition-only special memory concepts as defined in [special.mem.concepts]
+  template<typename S, typename I>
+  concept no-throw-sized-sentinel-for = // exposition only
+    no-throw-sentinel-for<S, I> &&
+    std::sized_sentinel_for<S, I>;
+
+  template<typename I>
+  concept no-throw-bidirectional-iterator = // exposition only
+    no-throw-forward-iterator<I> &&
+    std::bidirectional_iterator<I>;
```

Comment:
I think defining 7 exposition-only concepts (yes, technically two more are needed - for sentinels and forward iterators), of which only one is used in the algorithm signatures, is an overkill.

Perhaps I would only add `no-throw-sized-random-access-range` as the combination of sized and random access ranges with additional semantical requirements described by words.

Tagging @rarutyun for another opinion.